### PR TITLE
Fix issue #13

### LINKED ISSUE #13: Seems like i386 wireguard-tools package is broken
From maillist:
> Hello,
> 
> I've been trying to setup wireguard to experiment with it a bit and I've 
> run into a problem.  The i386 wireguard-tools package on the PPA for 
> Ubuntu Xenial doesn't actually contain any of the binaries, just 
> documentation.  You see this by checking the file listing at the end of 
> the build log here

> arm and ppc64 packages are also missing
> binaries.

Looking into it now. But from what moment we have ppc64 and arm builds activated for ppa?  
Pretty sure this is not default.

### PR DESCRIPTION
**Pull request by cryptofuture**
Should fix #13   
However needs an extra check on arm and ppc64.   
Works for i386

### CODE REVIEW

**Comment by EggieCode** [comment]
Maybe note in the changelog Jonathon of the mailing?

File: debian/rules [hunk 1]
```diff
@@ -9,10 +9,9 @@ export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 override_dh_auto_install-indep:
 	$(MAKE) -C wireguard-src/src DESTDIR=$(CURDIR)/debian/wireguard-dkms DKMSDIR=/usr/src/wireguard-$(DEB_VERSION_UPSTREAM)/ dkms-install
-	$(MAKE) -C wireguard-src/src/tools DESTDIR=$(CURDIR)/debian/wireguard-tools WITH_SYSTEMDUNITS=yes WITH_WGQUICK=yes WITH_BASHCOMPLETION=yes install
 
-override_dh_auto_build-arch:
```
Comment (zx2c4) [inline debian/rules:14]: Maybe you want to keep the build phase, in addition to adding the install phase? The two generally should be separate.

Comment (cryptofuture) [reply debian/rules:14]: Should work without, but its not a problem to add extra line with build-arch

Comment (zx2c4) [reply debian/rules:14]: Alright. In that case, then, let's just merge #15 and close this one?

Comment (EggieCode) [reply debian/rules:14]: Cool, maybe test it first on a private repo? That not everything crashes?

Comment (cryptofuture) [reply debian/rules:14]: Its the same and will produce the same result, just #15 more style correct.

Comment (zx2c4) [reply debian/rules:14]: Okay, in that case, can you close this #14, and give your +1 on #15's comments?

Comment (zx2c4) [reply debian/rules:14]: #wireguard on Freenode, by the way.

Comment (cryptofuture) [reply debian/rules:14]: Sorry, I can't join right now. I was intended to start work with another project actually... I don't think my help needed on irc anyway.

**Pull request closed without merging**